src/apispec/ext/marshmallow/__init__.py

```diff
@@ -53,21 +53,20 @@ class UserSchema(Schema):
 from apispec import BasePlugin
 from apispec.compat import itervalues
 from .common import resolve_schema_instance, make_schema_key, resolve_schema_cls
-from .openapi import OpenAPIConverter
+from .openapi import OpenAPIConverterMixin
 
 
 def resolver(schema):
-    """Default implementation of a schema name resolver function
-    """
+    """Default schema name resolver function that strips 'Schema' from the end of the class name."""
     schema_cls = resolve_schema_cls(schema)
     name = schema_cls.__name__
     if name.endswith("Schema"):
         return name[:-6] or name
     return name
 
 
-class MarshmallowPlugin(BasePlugin):
-    """APISpec plugin handling marshmallow schemas
+class MarshmallowPlugin(BasePlugin, OpenAPIConverterMixin):
```

Comment:
By convention, we put mixins before the base class because we usually want their methods to take precedence.

Reply:
Right. Fixed.